flutter-boilerplate/app_feup/lib/redux/actionCreators.dart

```diff
@@ -2,13 +2,32 @@ import 'package:redux_thunk/redux_thunk.dart';
 import '../model/AppState.dart';
 import 'actions.dart';
 import 'package:redux/redux.dart';
+import 'package:app_feup/controller/networking/NetworkRouter.dart';
 
-ThunkAction<AppState> login(name, password) {
+ThunkAction<AppState> login(username, password, persistentSession) {
   return (Store<AppState> store) async {
     //do requests, await futures
+    try {
+      final Map<String, dynamic> session = await NetworkRouter.login(username, password, persistentSession);
+      print(session);
+      store.dispatch(new SaveLoginDataAction(session));
+      if (session['authenticated'])
+        store.dispatch(fetchProfile());
```

Comment:
I think we should keep it like this for now because we don't have a homepage to show yet and the login system still needs testing, which this implementation facilitates

Reply:
[should do before approving] - After testing is done, we should have it redirect to the homepage before merging to develop